src/SignalR/server/SignalR/test/AuthHub.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Authorization;
+
+namespace Microsoft.AspNetCore.SignalR.Tests
+{
+    [Authorize]
+    class AuthHub : Hub
+    {
+    }
```

Comment:
So is the expectation that method-level auth doesn't play with endpoints or with the authZ middleware?